src/test/java/net/revelc/code/impsort/ImpSortTest.java

```diff
@@ -35,14 +36,14 @@
   public void testSort() throws IOException {
     Path p = Paths.get(System.getProperty("user.dir"), "src", "it", "plugin-test", "src", "test",
         "java", "net", "revelc", "code", "imp", "PluginIT.java");
-    new ImpSort(eclipseDefaults, false).parseFile(p);
+    new ImpSort(StandardCharsets.UTF_8.name(), eclipseDefaults, false).parseFile(p);
   }
 
   @Test
   public void testUnused() throws IOException {
     Path p =
         Paths.get(System.getProperty("user.dir"), "src", "test", "resources", "UnusedImports.java");
-    Result result = new ImpSort(eclipseDefaults, true).parseFile(p);
+    Result result = new ImpSort(StandardCharsets.UTF_8.name(), eclipseDefaults, true).parseFile(p);
```

Comment:
`parseFile` could pass the encoding to Result to use when saving

Reply:
done